.changeset/pre.json

```diff
@@ -0,0 +1,36 @@
+{
+  "mode": "pre",
+  "tag": "beta",
+  "initialVersions": {
+    "@swisspost/design-system-changelog-github": "1.0.2",
+    "@swisspost/design-system-components": "1.0.6",
+    "@swisspost/design-system-components-angular": "1.0.6",
+    "@swisspost/design-system-intranet-header": "3.0.6",
+    "@swisspost/design-system-demo": "5.2.0",
+    "@swisspost/design-system-documentation": "1.2.0",
+    "@swisspost/internet-header": "1.4.2-beta.0",
```

Comment:
Shouldn't it be `1.5.0-beta.1`?

Reply:
This lists the state when the "changeset pre enter beta" command was executed, meaning before the changesets for the beta were applied with `changeset version`. I'm not sure why changeset needs this list, but it reads like: that's what the repo looked like before you entered beta mode (at the moment it's only possible to enter a pre mode for the whole monorepo and not individual packages, that makes stuff like releasing only one package as pre-release a little difficult).